finetuner/tuner/keras/__init__.py

```diff
@@ -160,6 +160,9 @@ def fit(
         self.state = TunerState(num_epochs=epochs)
         self._trigger_callbacks('on_fit_begin')
 
+        # Check for early stopping
+        self.stop_training = False
```

Comment:
This attribute should not be declared inside the function, but rather in the base class.